open-craft/sprints
# [BB-1247] Prepare production

This tweaks production settings and adds CI for automating frontend and backend deployments.

## Testing instructions:
1. Visit https://sprints.opencraft.com and check that the dashboard is working as intended.
